lib/jekyll/plugin_manager.rb

```diff
@@ -27,7 +27,7 @@ def require_gems
       site.gems.each do |gem|
         if plugin_allowed?(gem)
           Jekyll.logger.debug("PluginManager:", "Requiring #{gem}")
-          require gem
+          Jekyll::External.require_with_graceful_fail([gem])
         end
```

Comment:
> is there still a point to having the require_gems method?

Absolutely! Code clarity. This whole thing is for GitHub after all, for their security team to ensure that non-whitelisted gems are not required. So if we can encapsulate this as its own method, it's easier to audit (and to understand that line's purpose).

> I also notice that require_plugin_files is still using require without the graceful_fail method.

Good point. You can fix that up too if you would like.